CircleciScripts/pre_start_simulator.sh

```diff
@@ -9,7 +9,7 @@ if [[ -z $test_device_id ]] ; then
   echo "Runtime: '${runtime}'"
 
   # Get the last alphabetical device (probably something in the iPhone X family, as of 2019-05-31)
-  devicetype=$( xcrun simctl list devicetypes iPhone | sort | tail -1 | sed 's/.*(//' | sed 's/).*//' | tr -d '[:space:]' )
+  devicetype=$( xcrun simctl list devicetypes "iPhone " | sort | tail -1 | sed 's/.*(//' | sed 's/).*//' | tr -d '[:space:]' )
```

Comment:
Without this change the list will contain iPod devices and might take some weird devicetype name.